REST Bindings/openapi.yaml

```diff
@@ -2372,6 +2397,9 @@ components:
         application/json:
           schema:
             $ref: '#/components/schemas/EPCISQueryDocument'
+        application/xml:
+          schema:
+            $ref: '#/components/schemas/EPCISXMLQueryDocument'
```

Comment:
why don't you reuse the EPCISQueryDocument directly?

Reply:
because the example won't come in an XML-way - there's no @context and many adjustments for xml would be required. also we can have seperate
```      externalDocs:
        description: |
          An EPCIS document that must be validated with the [XML Schema](../XSD/EPCglobal-epcis-query-2_0.xsd)
        url: ../XSD/EPCglobal-epcis-query-2_0.xsd ```